packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -454,14 +456,22 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
             addWidgetResizeHandler={this.props.addWidgetResizeHandler}
             removeWidgetResizeHandler={this.props.removeWidgetResizeHandler}
             unitController={this.props.unitController}
-            id='timegraph-chart'
+            id={this.props.outputDescriptor.id+'focusContainer'}
             layers={[
                 grid, this.chartLayer, selectionRange, this.chartCursors, this.arrowLayer, this.rangeEventsLayer
             ]}
         >
         </ReactTimeGraphContainer>;
     }
 
+    setFocus(): void {
+        if (document.getElementById(this.props.outputDescriptor.id+'focusContainer')) {
```

Comment:
It doesn't seem to work correctly when you open the same output for a different trace tab. I am guessing there are multiple elements with the same id.

Perhaps there is a way to find an element that is strictly a child of another specific element?

This comment applies to all outputs.